packages/applicant/src/components/partials/Header.tsx

```diff
@@ -1,199 +0,0 @@
-                        ></path>
-                      </svg>
-                    </>
-                  )}
-                  <span
-                    className="govuk-header__logotype-text"
-                    data-cy="cyGovLogoLink"
-                  >
-                    {' '}
-                    GOV.UK
-                  </span>
-                </span>
-              </a>
-            </Link>
-          </div>
-
-          <div className="govuk-header__content">
-            <a
-              href="/"
-              className="govuk-header__link govuk-header__link--service-name"
-            >
-              Find a grant
-            </a>
-          </div>
-        </div>
-      </header>
-      {/* SIGN IN/OUT */}
-      <div className="govuk-width-container govuk-!-text-align-right">
-        <p
-          className={`govuk-phase-banner__content govuk-!-margin-right-0 govuk-phase-banner__text ${styles['float-right']} govuk-!-font-size-19`}
-        >
-          {isUserLoggedIn ? (
-            <Link href={'/api/logout'}>
-              <a className="govuk-link govuk-link--no-visited-state">
-                Sign out
-              </a>
-            </Link>
-          ) : (
-            <a
-              href={process.env.LOGIN_URL}
-              className="govuk-link govuk-link--no-visited-state"
-            >
-              Sign in
-            </a>
-          )}
-        </p>
-      </div>
-      {/* BETA BLOCK */}
-      <div className="govuk-width-container ">
-        <div className="govuk-phase-banner">
-          <p className="govuk-phase-banner__content">
-            <strong className="govuk-tag govuk-phase-banner__content__tag">
-              BETA
-            </strong>
-            <span className="govuk-phase-banner__text">
-              This is a new service – your{' '}
-              <Link href={feedbackContent}>
-                <a
-                  className="govuk-link"
-                  target="_blank"
-                  data-cy="cyBetaFeedbackLinkBanner"
-                >
-                  feedback
-                </a>
-              </Link>{' '}
-              will help us to improve it.
-            </span>
-          </p>
-        </div>
-      </div>
-    </>
-  );
-};
-
-export default Header;
```

Comment:
This component is still being imported in Layout.tsx

Reply:
I actually have no idea where all these file deletions are coming from, I certainly didn't delete them manually.

Reply:
Deleted files resotred